client/store/login/actions/localLogin.ts

```diff
@@ -13,10 +14,17 @@ export const localLogin = createAsyncThunk<
     invitationUuid?: string
     history: RouteComponentProps['history']
   }
->('login/local', async ({ email, password, invitationUuid, history }) => {
-  await axios.post(`${ApiEndPoint.Auth.Login.local()}${invitationUuid ? `?invitationUuid=${invitationUuid}` : ''}`, {
-    email,
-    password,
-  })
-  history.push(BasePaths.Root())
+>('login/post/local', async ({ email, password, invitationUuid, history }, { dispatch }) => {
+  const response = await axios.post(
+    `${ApiEndPoint.Auth.Login.local()}${invitationUuid ? `?invitationUuid=${invitationUuid}` : ''}`,
+    {
+      email,
+      password,
+    }
+  )
```

Comment:
can we use axios params instead of concatenation?
Here is example: https://axios-http.com/docs/example